sdk/metric/aggregation/aggregation.go

```diff
@@ -164,3 +164,54 @@ func (h ExplicitBucketHistogram) Copy() Aggregation {
 		NoMinMax:   h.NoMinMax,
 	}
 }
+
+// ExponentialHistogram is an aggregation that summarizes a set of
+// measurements as an histogram with bucket widths that grow exponentially.
+type ExponentialHistogram struct {
+	// MaxSize is the maximum number of buckets to use for the histogram.
+	MaxSize int
```

Comment:
Based on [this comment](https://github.com/open-telemetry/opentelemetry-go/pull/4245/files#r1277626159) this value cannot exceed 2147483647, the max `int32` value.

```suggestion
	// MaxSize is the maximum number of buckets to use for the histogram.
	MaxSize int32
```

Reply:
If you read further, it should be able to accommodate expansions into full 64-bit width.  If anything it should be an int64, not an int32.

Reply:
While this does make sense, eventually, this number is going to be used in arithmetic involving slice indexes.  At some point, it will need to be converted into an `int`.  I plan on keeping the internal `int` to minimize the conversions necessary.